src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -3190,6 +3191,37 @@ private static VariableReference namedParam(String name, Type type) {
             namedParam("modifier", DataTypes.MODIFIER_TYPE));
     functions.add(new LibraryFunction("effect_modifier", DataTypes.EFFECT_TYPE, params));
 
+    params =
```

Comment:
`strings_modifier` that just returns a list of strings (for multistring) should probably also exist

Reply:
Done